### PR TITLE
Audience feed connector

### PR DESCRIPTION
**Pull request by PoulpiFr**
Please review =)

### CODE REVIEW

File: src/core/interfaces/mediarithmics/api/audience-feed-connector/AudienceFeedConnectorRequestInterface.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+export type UpdateType = 'UPSERT' | 'DELETE';
+
+export interface UserSegmentUpdateRequest {
+  feed_id: string;
+  session_id: string;
+  datamart_id: string;
+  segment_id: string;
+  user_identifiers: (UserPointIdentifierInfo | UserEmailIdentifierInfo | UserAccountIdentifierInfo | UserAgentIdentifierInfo)[];
+  ts: number;
+  type: UpdateType;
```
Comment (raabdesselam) [inline src/core/interfaces/mediarithmics/api/audience-feed-connector/AudienceFeedConnectorRequestInterface.ts]: `operation` not `type`

File: src/core/interfaces/mediarithmics/api/audience-feed-connector/AudienceFeedConnectorRequestInterface.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+export interface ExternalSegmentConnectionRequest {
+  feed_id: string;
+  session_id: string;
```
Comment (raabdesselam) [inline src/core/interfaces/mediarithmics/api/audience-feed-connector/AudienceFeedConnectorRequestInterface.ts]: You will get a null for session_id here, I didn't set the value of the session in the connection request

File: src/core/interfaces/mediarithmics/api/audience-feed-connector/AudienceFeedConnectorRequestInterface.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  datamart_id: string;
+  segment_id: string;
+}
+
+export interface ExternalSegmentCreationRequest {
+  feed_id: string;
+  session_id: string;
```
Comment (raabdesselam) [inline src/core/interfaces/mediarithmics/api/audience-feed-connector/AudienceFeedConnectorRequestInterface.ts]: same here

**Comment by PoulpiFr** [comment]
Closing this PR because it got disconnected from my fork when this repo went public.

The new PR is #25

**Pull request closed without merging**